### DISCUSSION TITLE
Can't clone from organisation I'm member of

### DISCUSSION THREAD

**Discussion by AlfieOnGit** [community/community#29859] [Other Feature Feedback, Questions, & Ideas] [answered]
So I'm trying to move my repos over to an organisation for simplicity. The organisation is the free version but I have member perms so I should be able to clone and create repos. I'm able to move my repos over and able to create new repos. Basically anything I do through github works but when I try and log in to my account with my IDE (IntelliJ) it treats it like I don't have any perms in the organisation. Either that or it doesn't understand how organisations work. If anyone knows or thinks they know that's up, please let me know! Thanks for your time!

**Comment by delosemre** [comment]
You can control your organization's third-party application access permissions. I'm not very knowledgeable about this, sorry.

**Reply by AlfieOnGit** [reply]
> You can control your organization's third-party application access permissions. I'm not very knowledgeable about this, sorry.

Thanks for the suggestion. Tried this but unfortunately not the issue

**Comment by jfullstackdev** [comment]
I'm not using IntelliJ but this one might help 

https://blog.jetbrains.com/idea/2020/10/clone-a-project-from-github/

**Reply by AlfieOnGit** [reply]
> I'm not using IntelliJ but this one might help
> 
> https://blog.jetbrains.com/idea/2020/10/clone-a-project-from-github/

This is how I would usually do it, yes, but it doesn't say how to do it through organisations. Thanks for the link tho

**Comment by jfullstackdev** [comment, marked as answer]
but maybe this one 

> If you want to use a PAT to access resources owned by an organization that uses SAML SSO, you must authorize the PAT. For more information, see "[About authentication with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on)" and "[Authorizing a personal access token for use with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)" in the GitHub Enterprise Cloud documentation.

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Reply by AlfieOnGit** [reply]
Thanks for the links. What worked was using the IDE to generate a personal access token instead of trying to log in through my account. Thank you so much, I can now sleep tonight xD
